app/policies/presentation_policy.rb

```diff
@@ -6,4 +6,12 @@ def create?
   def new?
     create?
   end
+
+  def update?
+    (person&.admin? || person&.presentations&.include?(record)).present?
```

Comment:
This how we are currently deciding if a person is signed in. We have different permissions for: not signed in, signed in and admin. If person is nil then they are not signed in.